External/FEXCore/Source/Interface/HLE/Thunks/Thunks.cpp

```diff
@@ -283,7 +283,14 @@ namespace FEXCore {
                 ERROR_AND_DIE_FMT("LoadLib: Failed to dlopen thunk library {}: {}", SOName, dlerror());
             }
 
-            const auto InitSym = fextl::string("fexthunks_exports_") + Name;
+            // Library names often include dashes, which may not be used in C++ identifiers.
+            // They are replaced with underscores hence.
+            auto InitSym = "fexthunks_exports_" + fextl::string { Name };
+            for (auto& c : InitSym) {
```

Comment:
Could use `fextl::string::replace` here I think?

Reply:
Maybe I'm overlooking a more useful overload, but `string::replace` seems to only replace characters *by position* rather than *by value*.

Reply:
Oh sorry, I was thinking `std::replace`

Reply:
Ohh, yes that would work. I thought it only replaced the first occurrence, but turns out it replaces all. Thanks!